cubeviz/tools/tests/test_smoothing_gui.py

```diff
@@ -0,0 +1,57 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+
+import pytest
+import numpy as np
+
+from qtpy import QtCore
+from glue.core import roi
+from cubeviz.tools.smoothing import SelectSmoothing
+
+from ...tests.helpers import (toggle_viewer, select_viewer, left_click,
+                      left_button_press, right_button_press, enter_slice_text,
+                      assert_all_viewer_indices, assert_slice_text)
+
+
+DATA_LABELS = ['018.DATA', '018.NOISE']
+
+
+@pytest.fixture(scope='module')
+def smoothing(cubeviz_layout):
+    cl = cubeviz_layout
+
+    sm = SelectSmoothing(cl._data, parent=cl)
+
+    return sm
+
+
+def assert_red_stylesheet(widget):
+    assert widget.styleSheet() == "color: rgba(255, 0, 0, 128)"
+
+
+def test_moment_maps_1(cubeviz_layout):
+    # Create GUI
+    sm = smoothing(cubeviz_layout)
+    sm.k_size.setText("1")
+    sm.combo.setCurrentIndex(0)
+    sm.component_combo.setCurrentIndex(0)
+
+    # Call calculate function and get result
+    np.warnings.filterwarnings('ignore')
+    sm.call_main()
+    print(cubeviz_layout._data, hasattr(cubeviz_layout._data, "container_2d"), cubeviz_layout._data.container_2d.component_ids())
+    # if hasattr(cubeviz_layout._data, "container_2d"):
+    #     for i in cubeviz_layout._data.container_2d.component_ids():
+    #         print(str(i))
+    smoothing_component_id = [str(x) for x in cubeviz_layout._data.container_2d.component_ids() if str(x).startswith('018.DATA_Smooth')][0]
```

Comment:
I think the failure is here and is related to some recent changes in `glue`, which should be fixed in the near future. It may require some changes to this test code as well.

cc @astrofrog